### PR TITLE
downgrade z-stream version used in 4.8 cluster-pool jobs

### PR DESCRIPTION
**Pull request by osherdp**
Since upgrade to ocp 4.8.3, jobs that use cluster-pool are failing to acquire clusters:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_assisted-service/2314/pull-ci-openshift-assisted-service-master-subsystem-aws/1421711050629189632
```
failed to wait for the created cluster claim to become ready: timed out waiting for the condition
```

Downgrading 4.8 temporally so that builds will get fixed for now.
/hold

### CODE REVIEW

**Review by filanov — APPROVED** [review]

**Comment by osherdp** [comment]
/hold
(to see if we'll be able first to debug current failures)

**Review by petr-muller — APPROVED** [review]

**Comment by petr-muller** [comment]
Let's merge this fast to unblock CI, and I'll make a little testing pool to investigate the problem outside of the production path

**Pull request merged**
